nagios/nagios-client/scripts/check_link_traffic.sh

```diff
@@ -7,14 +7,16 @@ SOCKPATH=/var/run/libtrap/munin_link_traffic
 pf="/tmp/`basename $0`-prevval"
 
 #getting current output of munin_link_traffic script
+headers="`nc -U "$SOCKPATH" </dev/null 2>/dev/null| awk 'NR==1' | tr ',' ' '`"
```

Comment:
This is very bad solution - to read data twice...
Headers and values MUST be read at once using one nc process.